_posts/dread/2024-03-10-export-romfs.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "How to extract your Metroid Dread RomFS"
+tags: dread
+---
+You can either extract your RomFS using nxdumptool from your Switch, or using Ryujinx from an image (.nsp or .xci). We'll show you how to do it using Ryujinx.
+
+First, download and setup Ryujinx. Make sure you follow their start-up guide after installation. You can download Ryujinx here: https://ryujinx.org/download
+
+Once you've set it up, navigate to **Options -> Settings** to add the directory with your game image. Your game should be shown in Ryujinx now.
+
+![](/assets/guides/romfs/ryujinx1.png)
```

Comment:
all ambeded images should get alt text for accessiblity.

Reply:
out of scope; open an issue as it applies to the existing site as well